src/json_reporter.cc

```diff
@@ -77,6 +77,8 @@ bool JSONReporter::ReportContext(const Context& context) {
   std::string walltime_value = LocalDateTimeString();
   out << indent << FormatKV("date", walltime_value) << ",\n";
 
+  out << indent << FormatKV("host_name", context.sys_info.name) << ",\n";
```

Comment:
1. We are sure that this won't contain `\\`? (see `executable_name`)
2. Do we want to print `"Unable to Get Host Name"`?2
3. I wonder if this should be after the `executable_name`.

Reply:
2. No. We should check and only output if it's available.

Reply:
1. Yes, I checked the output, it just outputs the hostname for windows without the `\\`
3. I am not sure about this one, it can be done, but since the output being in json only, is it desirable?

Reply:
Aren't we still always printing it?

Reply:
Looks about right other than this.

Reply:
are you talking about the 2nd point?
I have placed an empty string incase it fails to fetch the host name